imports/ui/components/GeoSuggest.jsx

```diff
@@ -33,11 +33,16 @@ export class GeoSuggest extends React.Component {
   };
 
   render() {
+    const searchOptions = {
+      location: new google.maps.LatLng(49.290338, -123.134112),
+      radius: 100
```

Comment:
One suggestion is just to add a comment before searchOptions variable saying ```These options will bias the autocomplete predictions toward <your-latlng> with a radius of 100 meters and limit the results to addresses only``` or something like that so we remember it is in meters and what that latlng is set to.

Reply:
Second that! Good comment to put :D

Reply:
Added!